### PR TITLE
Support for currencies other than MXN

### PR DESCRIPTION
**Pull request by cavalle**

### CODE REVIEW

File: cfdi.go [hunk 1]
```diff
@@ -187,13 +189,9 @@ func validateSupport(inv *bill.Invoice) error {
...
 	}
 
-	if inv.Currency != currency.MXN {
-		errs["currency"] = ErrNotSupported
-	}
-
 	for _, l := range inv.Lines {
-		if l.Item.Currency != currency.CodeEmpty && l.Item.Currency != currency.MXN {
-			errs["line currency"] = ErrNotSupported
+		if l.Item.Currency != currency.CodeEmpty && l.Item.Currency != inv.Currency {
```
Comment (samlown) [inline cfdi.go:193]: I don't think this is necessary as GOBL will always ensure that the item's currency matches that of the document.

File: cfdi.go [hunk 2]
```diff
@@ -284,6 +282,16 @@ func lookupTipoDeComprobante(inv *bill.Invoice) string {
 	return code.String()
 }
 
+func tipoCambio(inv *bill.Invoice) string {
+	for _, r := range inv.ExchangeRates {
```
Comment (samlown) [inline cfdi.go:286]: This method might be a bit nice to use: https://github.com/invopop/gobl/blob/main/currency/exchange_rate.go#L62

**Review by samlown — APPROVED** [review]
Looking good! A few minor points to consider.

File: cfdi.go (hunk 1 above)
Comment (cavalle) [reply cfdi.go]: Addressed on be25203. Thanks!

File: cfdi.go (hunk 2 above)
Comment (cavalle) [reply cfdi.go]: Nice one! Addressed on b3427b3

**Pull request merged**